project.php

```diff
@@ -190,13 +195,11 @@ function get_bookmark_form(): string
         <form method='POST' style='display: inline;'>
         <input type='hidden' name='id' value='$project->projectid'>
         <input type='hidden' name='bookmark' value='$bookmark_value'>
-        <a
-            href='#'
-            onClick='this.closest("form").submit();'
+        <button
             aria-label='$label'
             title='$tooltip'
-            style='font-size: 60%'
-        ><i class='$icon' aria-hidden='true'></i></a>
+            style='font-size: 60%; border: none; background: none;'
```

Comment:
I don't know how much of this is that it's different, but I like that the current version looks like a link (color, and cursor change) to indicate that it's something users can interact with. I don't think it has to be a link, but the link-like behavior seems advantageous for discoverability.

I'm curious what other reviewers think.

Reply:
We can make the cursor change to a pointing hand over it and change its colour.

Reply:
Yeah, we need something. I would never have clicked it unless I knew I was supposed to.

Reply:
I think we missed changing the color -- I'd make it `@link-color`. I'm not worried about it always having the same color even after clicked, but I think it needs to be more obvious that it's something you can click.

Reply:
We could also make the backgound change colour when the cursor approaches it like Slack does. I'll do that as well then we can choose which options look best.